index.html

```diff
@@ -76,20 +76,24 @@
 
   <p class="note">The user may enable such preference, if made available by the user agent, due to high data transfer costs, slow connection speeds, or other reasons.</p>
 
-  #### <code><dfn>Save-Data</dfn></code> Request Header Field
+  #### <code><dfn>Sec-CH-Save-Data</dfn></code> Request Header Field
 
   The <a>SaveData</a> request header is a Client Hint [[CLIENT-HINTS]]. The header's value is an `sh-list` [[STRUCTURED-HEADERS]] which contains `token`s, indicating the user agent's preference for reduced data usage.
 
   <pre class="nohighlight">
-    Save-Data = sh-list
+    Sec-CH-Save-Data = sh-list
   </pre>
 
+  <div class="note">
+    NOTE: The old name for this header (`Save-Data`) will continue to be sent by default until further notice. The new name (`Sec-CH-Save-Data`) will not be sent unless specifically requested in the normal client hints flow.
```

Comment:
Maybe "Browsers that send the `Save-Data` header by default are likely to continue to do that for backward compatibility purposes. The new header name..."